Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,18 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
+
+export const CardContent = () => {
+  return (
+    <div className="banner-text">
+      <h3 className="banner-heading">Get Started with React</h3>
+      <p className="text-section">
+        React makes it painless to create interactive UIs. Design simple views
+        for each state in your application
+      </p>
+      <a href="https://reactjs.org" className="react-url">
```

Comment:
Go ahead and move your anchor tag up above your h3 tag here so you can wrap all of the content inside the banner-text <div> with the link like the README file wants for this assignment :)